CMSgov/design-system
# [WNMGDS-1353] Design Tokens Sass Conversion

## Summary

Includes additions to the design system for token scss generation including helper functions and cli setup to be utilized for other conversion types (sketch, etc..)

### Added

packages/design-system-tokens subfolder includes everything needed

## How to test

`yarn build themes scss` `yarn build tokens scss` `yarn build tokens csv` `yarn build themes csv` from the tokens subfolder to create output scss, csv in the './dist/' folder there.  Feel free to play around with importing tokens typescript and using them in code to validate correct signatures appear. 

Current Styles which I've gotten as close to the original colors as I *think* could be done, @line47 you'll have to let me know where substitutions should be made

https://design.cms.gov/
vs
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1353-tokens-to-sass/core/
